docs/IntentReader.md

```diff
@@ -1,62 +1,92 @@
 # FCA: Intent Reader
 
-TODO: add intro
+IntentReader is a powerful feature of FCA which accepts an input command or intent and performs various actions/tasks based on the command received. This is possible with the help of multiple  handlers implemented in FCA. IntentReader is the decision maker which would choose which handler to call based on the input command. Input command/intent is used interchangeably throughout this document.
 
-## Table of Contents
+For eg: if the task is "clearAllListeners", it will be calling the "clearEventListeners" handler to clear all the eventListeners and the event history. Similarly if the task is "callMethod", "CallMethodHandler" will be called which is used to invoke any apis.
```

Comment:
Being picky here but can we say "For example" instead of For eg:

Reply:
Replaced with For example